tests/cypress/e2e/features/ground_truth_jobs.js

```diff
@@ -323,23 +353,52 @@ context('Ground truth jobs', () => {
             cy.get('.cvat-objects-sidebar-show-ground-truth').filter(':visible').click();
 
             cy.goCheckFrameNumber(groundTruthFrames[0]);
-            checkConflicts('warning', 1);
+            checkConflicts('error', 2);
+            checkHighlight(1);
 
             cy.goCheckFrameNumber(groundTruthFrames[1]);
-            checkConflicts();
+            checkConflicts('warning', 1);
 
             cy.goCheckFrameNumber(groundTruthFrames[2]);
+            checkConflicts();
```

Comment:
checkConflicts does not have default value for `type` and `amount`

Reply:
Added